src/cozy/wrappers/button-extension/button-extension.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import {
+    AngularWrapperComponent,
+    AngularWrapperProps,
+} from '../angular-wrapper.component';
+// @ts-ignore
+import ButtonExtension from './button-extension.jsx';
+
+@Component({
+    selector: 'app-button-extension',
+    templateUrl: '../angular-wrapper.component.html',
+    encapsulation: ViewEncapsulation.None,

```

Comment:
for my curiosity, when does that mean? This `encapsulation` stuff?

Reply:
This is about CSS encapsulation. By default a ShadowDom is emulated and parent CSS cannot affect child CSS, this would prevent us to import cozy-ui style in a single place. Instead we would have to import them in each wrapped component.

https://github.com/cozy/cozy-pass-web/blob/93e2d903f190b6c5570c4d5b134dac53184adbb8/src/scss/cozy.scss#L1-L4